cogs/tmpc.py

```diff
@@ -434,24 +472,80 @@ async def nomod(self, ctx: Context):
         """
         global moderator
 
-        document = await self.check_tmpc_channel(ctx, is_mod=True)
+        document = await self.check_tmpc_channel(ctx.author, ctx.channel.id, is_mod=True)
         await document.voice.set_permissions(moderator.item, overwrite=None)
         await document.chat.set_permissions(moderator.item, overwrite=None)
         embed: Embed = Embed(title="No Mod",
                              description=f"Mods can't see or join this channel anymore")
         await ctx.reply(embed=embed)
 
-    async def check_tmpc_channel(self, ctx: Context, is_mod: bool = False, everyone: bool = False) \
+    @tmpc.command(pass_context=True,
+                  brief="A user to your temp channel",
+                  help="Default add user to the temp channel, for gaming channel the first argument must be 'False'!\n"
+                       "Member must be invited by mentions.\n"
+                       "Limitations:\n"
+                       "- This command can be used 3/hour/user\n"
```

Comment:
Reason for this limitation?

Reply:
Yeah this shall prevent abusing.
Otherwise a User can recreate a tempchannel and force join any other member or force invite all members.

Of cause this command can repeated over the day but not so massive and we would send the abuser into time-out.

Reply:
Ah ok then it is clear thx